synapse/config/registration.py

```diff
@@ -99,6 +99,7 @@ def read_config(self, config, **kwargs):
         self.trusted_third_party_id_servers = config.get(
             "trusted_third_party_id_servers", ["matrix.org", "vector.im"]
         )
+        self.account_threepid_delegate = config.get("account_threepid_delegate")
```

Comment:
this never appears to be read as anything other than a boolean - is proper support for it on its way?